src/workers/atlas_strategies/challenge_resolution_strategy.js

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+/** @abstract */
+export default class ChallengeResolutionStrategy {
+  get challengePullingInterval() {
```

Comment:
I would stick with `workerInterval` as in the Hermes upload strategy. 

On the other hand 👍 for using a property instead of a function. Could you update the Hermes strategy to also use one?

Reply:
Also: the name of the class should be `AtlasChallengeParticipationStrategy`